src/current/_includes/metrics-usage/tenant.consumption.cross_region_network_ru.md

```diff
@@ -0,0 +1 @@
+The number of RUs that were consumed due to cross-region networking. Correlate these metrics with Request Units (RUs). <a href="https://www.cockroachlabs.com/docs/cockroachcloud/plan-your-cluster-serverless#multi-region-clusters">Learn more</a>.
```

Comment:
Suggest changing "learn more" to something more descriptive like "For more information about multi-region clusters and networking, see 'Multi-region clusters'" (or whatever the section name is in the linked doc)